html/img_import/get_images.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+ini_set('display_errors', 1);
+require 'credentials.inc.php';
```

Comment:
this should be "require '../credentials.inc.php'" so that there's only one "credentials.inc.php" file in the repo. And this file is not uploaded to the Github repo.